server/views/pages/index.njk

```diff
@@ -54,6 +54,17 @@
             "id": "view-timetable"
           }) }}
         </li>
+        {% if showFeatureSessionManagement %}
+          <li class="govuk-grid-column-one-third card-group__item">
+          {{ card({
+            "href": "/date-management",
```

Comment:
```suggestion
            "href": "/block-visit-dates",
```
Unless it's specified on the ticket, I'd suggest this alternative URL as GDS guidance is for URLs to align to page titles.